<ui version="4.0" >
 <class>SearchReplace</class>
 <widget class="QWidget" name="SearchReplace" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frameSearch" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>2</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="bClose" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Close search/replace panel</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../../src/lib/images/Edyuk.qrc" >
          <normaloff>:/closeall.png</normaloff>:/closeall.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bRefresh" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Refresh search underlying context (as an attempt to correct search behavior)</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../../src/lib/images/Edyuk.qrc" >
          <normaloff>:/reload.png</normaloff>:/reload.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string> Find :</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFind" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>120</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Text or pattern to search for</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bNext" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Find next occurence</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../../src/lib/images/Edyuk.qrc" >
          <normaloff>:/down.png</normaloff>:/down.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bPrevious" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Find previous occurence</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../../src/lib/images/Edyuk.qrc" >
          <normaloff>:/up.png</normaloff>:/up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cbWords" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Words</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCursor" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Cursor</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSelection" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbHighlight" >
        <property name="text" >
         <string>Highlight all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbRegExp" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Regexp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCase" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Case</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameReplace" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>2</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbReplace" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Replace :</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leReplace" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>168</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>1200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Replacement text</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>139</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cbPrompt" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Prompt on replace</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbReplaceAll" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Replace all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEscapeSeq" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Escape sequences</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../src/lib/images/Edyuk.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cbReplace</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameReplace</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>46</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bClose</sender>
   <signal>clicked()</signal>
   <receiver>SearchReplace</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>11</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>202</x>
     <y>-27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
